src/log.ts

```diff
@@ -7,22 +7,23 @@ import {
   LogLevels,
   LogRecord,
   red,
-  reset,
+  resolve,
   yellow,
 } from "../deps.ts";
 
-const DEBUG_LEVEL = "DEBUG";
-const QUIET_LEVEL = "ERROR";
-const DEFAULT_LEVEL = "INFO";
+import { version } from "./version.ts";
 
-const DEFAULT_HANDLER = "format_fn";
+const logFile = "./eggs-debug.log";
 
-export class ConsoleHandler extends BaseHandler {
+let masterLogRecord = "";
+let detailedLog = false;
+
+class ConsoleHandler extends BaseHandler {
   format(record: LogRecord): string {
     let msg = "";
     switch (record.level) {
       case LogLevels.DEBUG:
-        msg += green("[DBG]");
+        msg += green("[DEBUG]");
```

Comment:
Consider making this grey as debug is a background detail. Green should be used for more success related messages imo.